package.json

```diff
@@ -25,6 +25,9 @@
 		"test:browser:dev": "pnpm --filter @gradio/app test:browser:dev",
 		"ci:publish": "pnpm publish --no-git-checks --access public -r",
 		"ci:version": "changeset version && pnpm i --lockfile-only",
+		"storybook": "storybook dev -p 6006",
+		"build-storybook": "storybook build",
+		"chromatic": "chromatic --exit-zero-on-changes",
```

Comment:
If we did something custom, I guess we'd need to tell chromatic where out storybook dist ended up.